### PR TITLE
[FSSDK-9494] Quality of life updates picked from #210

### PR DESCRIPTION
**Pull request by mikechu-optimizely**
## Summary
- General updates cherry-picked from #210 to improve developer experience
- Brings in #187 

## Test plan
- All existing tests should pass

## Issues
- FSSDK-9494

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "2.9.2",
   "description": "React SDK for Optimizely Feature Experimentation, Optimizely Full Stack (legacy), and Optimizely Rollouts",
   "homepage": "https://github.com/optimizely/react-sdk",
+  "repository": "https://github.com/optimizely/react-sdk",
```
[the remark this answers is not in the document]
Comment (mikechu-optimizely) [reply package.json:6]: I still see some transitive deps that use 4.0.0 😢  

We need to upgrade those next.

[the remark this answers is not in the document]
Comment (mikechu-optimizely) [reply package.json:6]: Grrrr tough-cookie@4.0.0 is needed by jsdom@^16.4.0 which is used by jest-environment-jsdom@^26.6.2

[the remark this answers is not in the document]
Comment (mikechu-optimizely) [reply package.json:6]: This requires a heavier upgrade than a quality of life.  Moving to the next PR.

File: package.json [hunk 2]
```diff
@@ -33,9 +34,10 @@
     "access": "public"
   },
   "dependencies": {
-    "@optimizely/optimizely-sdk": "^5.0.0-beta2",
+    "@optimizely/optimizely-sdk": "5.0.0-beta2",
     "hoist-non-react-statics": "^3.3.0",
     "prop-types": "^15.6.2",
+    "tough-cookie": "4.1.3",
```
Comment (mikechu-optimizely) [inline package.json:40]: Hmmm I'm now questioning this addition as an attempt to solve the Prisma CVE notification.  I don't think making this a direct dep is a solution.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (mikechu-optimizely) [reply package.json:6]: 😔

**Review by raju-opti — APPROVED** [review]
✔️

**Pull request merged**